acceptance/bundle/deploy/fail-on-active-runs/script

```diff
@@ -0,0 +1,7 @@
+trace $CLI bundle deploy # We deploy the bundle first to create a job
+trace errcode $CLI bundle deploy --fail-on-active-runs # We deploy the bundle again to check that the deploy is failing if the job is running
```

Comment:
Could you move the comments to their own line? Reads a bit nicer from top to bottom.

Reply:
Done